### ISSUE TITLE
Can not create Password Database on Linux

### ISSUE THREAD

**Issue by Jheschl17** [authpass/authpass#131]
Using the authpass version in the Linux PPA (1.7.4+1397) I can not create a new password database. When I click on "New to KeePass? Create New Password Database", enter a database name and master password and click "Create Database", nothing happens.

According to the logs printed when starting authpass with the terminal, the dynamic library `./libargon2_ffi_plugin.so` could not be loaded.

Full error message:
```
flutter: 2020-08-25 11:08:45.373874 FINER analytics - event(track, createFile, , null) - parameters: {}
flutter: 2020-08-25 11:08:45.374786 FINER analytics - event(track, saveCount, generator=AuthPass, 1) - parameters: {}
flutter: 2020-08-25 11:08:45.375221 FINEST kdbx_var_dictionary - Reading VarDictionary 1.0
flutter: 2020-08-25 11:08:45.375467 FINER kdbx.header - Writing header HeaderFields.CipherID (16)
flutter: 2020-08-25 11:08:45.375495 FINER kdbx.header - Writing header HeaderFields.CompressionFlags (4)
flutter: 2020-08-25 11:08:45.375516 FINER kdbx.header - Writing header HeaderFields.MasterSeed (32)
flutter: 2020-08-25 11:08:45.375537 FINER kdbx.header - Writing header HeaderFields.EncryptionIV (16)
flutter: 2020-08-25 11:08:45.375558 FINER kdbx.header - Writing header HeaderFields.KdfParameters (139)
flutter: 2020-08-25 11:08:45.375577 FINER kdbx.header - Writing header HeaderFields.EndOfHeader (0)
flutter: 2020-08-25 11:08:45.375641 FINEST kdbx_var_dictionary - Reading VarDictionary 1.0
flutter: 2020-08-25 11:08:45.375688 FINE key_encrypter_kdf - Must be using argon2
flutter: 2020-08-25 11:08:45.375711 FINEST kdbx_argon2_ffi - Starting argon2
flutter: 2020-08-25 11:08:45.392036 FINE kdbx_argon2_ffi - Finished argon2 in 16ms
[ERROR:flutter/lib/ui/ui_dart_state.cc(171)] Unhandled Exception: Exception: Invalid argument(s): Failed to load dynamic library (./libargon2_ffi_plugin.so: cannot open shared object file: No such file or directory)
#0      _open (dart:ffi-patch/ffi_dynamic_library_patch.dart:11)
#1      new DynamicLibrary.open (dart:ffi-patch/ffi_dynamic_library_patch.dart:20)
#2      Argon2FfiFlutter._loadLib (package:argon2_ffi_base/src/argon2_ffi_impl.dart:60)
#3      new Argon2FfiFlutter (package:argon2_ffi_base/src/argon2_ffi_impl.dart:84)
#4      FlutterArgon2.singleton (package:authpass/bloc/kdbx_argon2_ffi.dart:12)
#5      FlutterArgon2.singleton (package:authpass/bloc/kdbx_argon2_ffi.dart)
#6      FlutterArgon2.runArgon2 (package:authpass/bloc/kdbx_argon2_ffi.dart:31)
#7      _IsolateConfiguration.apply (package:flutter/src/foundation/_isolates_io.dart:81)
#8      _spawn.<anonymous closure> (package:flutter/src/foundation/_isolates_io.dart:88)
#9      _spawn.<anonymous closure> (package:flutter/src/foundation/_isolates_io.dart:87)
#10     Timeline.timeSync (dart:developer/timeline.dart:163)
#11     _spawn (package:flutter/src/foundation/_isolates_io.dart:85)
#12     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:304)
#13     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168)
```

### System Details:
Operating System: Kubuntu 20.04
KDE Plasma Version: 5.18.5
KDE Frameworks Version: 5.68.0
Qt Version: 5.12.8
Kernel Version: 5.4.0-42-generic
OS Type: 64-bit
Processors: 12 × AMD Ryzen 5 2600X Six-Core Processor
Memory: 15,6 GiB of RAM

**Comment by hpoul** [comment]
Thanks for the bug report, can you try a `ldd /usr/lib/authpass/lib/libargon2_ffi_plugin.so` and paste the output here? maybe there is some library missing from the dependencies.

**Comment by hpoul** [comment]
@Jheschl17 nvm i think I found the problem.. it looks in the same directory as the binary, not the `lib` sub directory .. as a workaround you should be able to do a `sudo cp /usr/lib/authpass/lib/libargon2_ffi_plugin.so /usr/lib/authpass/` until it is fixed

**Comment by hpoul** [comment]
@Jheschl17 I have now fixed this, and upload a new version to ppa (1.7.5+1422) please try to update and let me know if it works for you. thanks!
